libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -59,4 +64,15 @@ export class BookSearchComponent implements OnInit {
       this.store.dispatch(clearSearch());
     }
   }
+
+  openSnackBar(book: Book) {
+    const snackBarRef = this.snackBar.open(book.title + ' is added to list', 'Undo');
```

Comment:
Also, snackbar does not have a duration, here... It will stay open forever if the user does not interact with it. Usually, snackbars close on their own after a few seconds.